packages/cactus-test-cmd-api-server/src/test/typescript/integration/remote-plugin-imports.test.ts

```diff
@@ -27,86 +27,101 @@ import {
   PluginImportType,
 } from "@hyperledger/cactus-core-api";
 
-test("NodeJS API server + Rust plugin work together", async (t: Test) => {
-  const vaultTestContainer = new VaultTestServer({});
-  await vaultTestContainer.start();
+const testCase = "NodeJS API server + Rust plugin work together";
 
-  const ci = await Containers.getById(vaultTestContainer.containerId);
-  const vaultIpAddr = await Containers.getContainerInternalIp(ci);
-  t.comment(`Container VaultTestServer has IPv4: ${vaultIpAddr}`);
+describe(testCase, () => {
+  let vaultTestContainer: VaultTestServer;
+  let apiServer: ApiServer;
+  let pluginContainer: any;
 
-  test.onFinish(async () => {
-    await vaultTestContainer.stop();
-    await vaultTestContainer.destroy();
+  afterAll(async () => {
+    vaultTestContainer.stop();
+    vaultTestContainer.destroy();
+    apiServer.shutdown();
+    pluginContainer.stop();
```

Comment:
@Leeyoungone Please await for the stop to finish before calling destroy. Otherwise it might be a race condition to run the two concurrently.